pkg/providers/merge/worker.go

```diff
@@ -84,28 +85,58 @@ func (m *merge) startPolling() {
 				continue
 			}
 
-			var job *AutoMerge
-			for _, model := range jobs {
-				if model.Started {
-					job = model
-				}
-			}
-			if job == nil {
-				job = jobs[0]
-				m.startJob(job)
+			// var job *AutoMerge
+			// for _, model := range jobs {
+			// 	if model.Started {
+			// 		job = model
+			// 	}
+			// }
+			// if job == nil {
+			// 	job = jobs[0]
+			// 	m.startJob(job)
+			// }
+
```

Comment:
This code maybe was wrong?

the true like this?
```go
            var job *AutoMerge
            for _, model := range jobs {
                if model.Started {
                    job = model
                    break
                }
            }
            if job == nil {
                job = jobs[0]
                m.startJob(job)
            }
```

Reply:
@you06 PTAL :smile:

Reply:
Good catch!

Reply:
Fortunately, I was copying a pointer here instead of take address, but it should be break!